services/director-v2/src/simcore_service_director_v2/modules/comp_scheduler/__init__.py

```diff
@@ -1,7 +1,38 @@
+import logging
+from collections.abc import Callable, Coroutine
+from typing import Any, cast
+
 from fastapi import FastAPI
+from servicelib.logging_utils import log_context
 
+from . import _scheduler_factory
 from ._base_scheduler import BaseCompScheduler
-from ._task import on_app_shutdown, on_app_startup
+
+_logger = logging.getLogger(__name__)
+
+
+def on_app_startup(app: FastAPI) -> Callable[[], Coroutine[Any, Any, None]]:
+    async def start_scheduler() -> None:
+        with log_context(
+            _logger, level=logging.INFO, msg="starting computational scheduler"
+        ):
+            app.state.scheduler = scheduler = await _scheduler_factory.create_from_db(
```

Comment:
Are there supposed to be two `=`s on this line? 😅

Reply:
yes :rofl: